### DISCUSSION TITLE
Can Philips Wiz work in an IPv6 Only environment

### DISCUSSION THREAD

**Discussion by owendelong** [sbidy/wiz_light#140] [Q&A]
Will the bulbs (and this software) work in an IPv6-only network?

**Comment by owendelong** [comment]
I'm willing to work on updating this software if the bulbs support it and the software doesn't.

**Comment by n-engelhardt** [comment]
I looked at the app and they don't list an IPv6 address for the device. Unless it's getting one but isn't being listed in the app, it doesn't look like these devices support IPv6.

**Reply by sbidy** [reply]
From my point of view the integration is able to use IPv6 addresses but the WiZ FW (the ESP inside) doesn't.
Maybe the colleges from WiZ will do an FW update in the future.

**Comment by owendelong** [comment]
The ESP is certainly capable of using IPv6, the question is whether or not Philips chose to implement this capability.

Philips mostly seems unable to figure out how to spell IPv6 so far, which is very disappointing and renders this product
unusable in my environment.

Owen
